src/main/java/com/uid2/shared/Utils.java

```diff
@@ -184,4 +183,22 @@ public static <T1, T2> int upperBound(List<T1> list, T2 value, BiPredicate<T2, T
         }
         return first;
     }
+
+    //can make this as an config in admin/core service config
+    public static final String SiteSpecificDataSubDirPath = "sites/";
```

Comment:
I think in Java only classes should be named with PascalCase.
In the design spec I was going with /site rather than /sites so far, following more of a db table naming convention for "table" names to be singular.

Reply:
ok will correct them after you have finalised the path to the site data directory paths and i can update this

Reply:
Will do this in the new UID2-575 ticket branch later after UID2-598 is done